lib/src/main/res/values/strings.xml

```diff
@@ -1,44 +1,53 @@
+    <string name="nextcloud_files_app_not_installed_action">Install</string>
 
-    <string name="nextcloud_files_app_account_not_found_title">Error</string>
-    <string name="nextcloud_files_app_account_not_found_message">Account not found in Nextcloud Files app</string>
+    <string name="nextcloud_files_app_account_not_found_title">Account not found</string>
+    <string name="nextcloud_files_app_account_not_found_message">The requested account was not found in Nextcloud Files app</string>
+    <string name="nextcloud_files_app_account_not_found_with_account_message" formatted="true">The account %1$s was not found in Nextcloud Files app</string>
 
-    <string name="nextcloud_files_app_account_permission_not_granted_title">Error</string>
-    <string name="nextcloud_files_app_account_permission_not_granted_message">You have to accept the requested permissions to use the single sign-on feature.</string>
+    <string name="nextcloud_files_app_account_permission_not_granted_title">Permission not granted</string>
+    <string name="nextcloud_files_app_account_permission_not_granted_message">You have to accept the requested permissions to use the Single Sign On feature.</string>
 
-    <string name="nextcloud_http_request_failed_title">Error</string>
+    <string name="nextcloud_http_request_failed_title">Request failed</string>
     <string name="nextcloud_http_request_failed_message" formatted="true">HTTP request failed with HTTP status-code: %1$d</string>
 
     <string name="nextcloud_unsupported_method_title" translatable="false">Error</string>
-    <string name="nextcloud_unsupported_method_message" translatable="false">The requested HTTP Method is not supported yet! Please contact the developers of the Android Single-On Library for Android</string>
+    <string name="nextcloud_unsupported_method_message" translatable="false">The requested HTTP Method is not supported yet! Please contact the developers of the Single Sign On Library for Android</string>
+    <string name="nextcloud_unsupported_method_action">Contact</string>
 
-    <string name="nextcloud_invalid_request_url_title" translatable="false">Error</string>
+    <string name="nextcloud_invalid_request_url_title" translatable="false">Invalid URL</string>
     <string name="nextcloud_invalid_request_url_message" translatable="false" formatted="true">The requested URL is invalid: %1$s</string>
 
-    <string name="nextcloud_files_app_not_supported_title">Error</string>
-    <string name="nextcloud_files_app_not_supported_message" formatted="true">Your Nextcloud Files app currently does not support the single sign on feature. Please update it: %1$s</string>
+    <string name="nextcloud_files_app_not_supported_title">Update Nextcloud Files app</string>
+    <string name="nextcloud_files_app_not_supported_message">Your Nextcloud Files app is outdated and does not yet support the Single Sign On feature.</string>
```

Comment:
Better would be: "Your Nextcloud Files app is an older version which does not yet support the Single Sign On feature."

"Outdated" sounds like it's generally bad, older version is more specific.

Reply:
Thanks, updated!